reconfiguration/src/reconfiguration_handler.cpp

```diff
@@ -197,7 +197,7 @@ BftReconfigurationHandler::BftReconfigurationHandler() {
   std::ifstream key_content;
   key_content.open(operatorPubKeyPath);
   if (!key_content) {
-    LOG_ERROR(getLogger(), "unable to read the operator public key file");
```

Comment:
This could be an error depending how the operator public key file is installed on a machine. If the file is pre-installed or there's no mechanism to obtain it dynamically (and recover) an error could be more useful to draw in attention.

Reply:
agreed. We need to add additional checks here